decentraland/unity-explorer
# feat: logout & system menu

Implements a profile widget and a system menu accessible from the explore menu.

The profile widget displays:
- Current user name + address #{^4...}
- Avatar's face256 snapshot

From the system menu you can:
- Exit the application
- Logout, then will restart the startup flow
- Open privacy policy link
- Open terms link

Other UX improvements:

- The zoom & panning has been enabled on authentication screen so you can properly preview avatars that get out of screen
- Prevent avatar preview from clipping on the top on authentication screen
- Reverted direction of rotation on avatar preview
- Explore menu icon moved next to the map

# How to test?

1. Login with an account A
2. Check your avatar is correctly previewed with no clipping
3. Check you can zoom in/out with scroll and pan the camera dragging right click button
4. Observe that the initialization flow (authentication+loading screens) works as expected
5. Open the explore panel from the top left button
6. Observe that the profile widget contains the correct information. If you do not claimed your name, the format should be: `name#last4CharactersOfAddress`
7. Click on the profile widget on the top right
8. Test the different options of the menu
9. Select logout option. Observe that the authentication screen is opened
10. Login with another account B
11. Observe that the flow works as expected with your new user
12. Observe that the avatar now has the wearables of the account B
13. Quit the application through the system menu by clicking on your avatar's capsule in the explore panel. The app should close correctly without any crashes
